quay/quay
# Use Redhat contact information for Quay.io

### Description of Changes

Remove old terms and privacy statements. Update contact information.

**NOTE**: Do **NOT** deploy this before March 1, 2020.

#### Changes:

* Remove deprecated Terms, Privacy, and Contact pages from source code.
* Update contact link in multiple pages to use the same link as shown in the footer.

#### Issue:

* https://issues.redhat.com/browse/PROJQUAY-160
* https://issues.redhat.com/browse/PROJQUAY-161
* https://issues.redhat.com/browse/PROJQUAY-162

**TESTING**

* Ensure links use the correct URL as mentioned in the changes/description and associated Jira tickets in the footer.
* Ensure the correct links are used to contact support in all changed files.
* Ensure attempting to access `/contact` `/privacy` and `/tos` results in a 404 or otherwise does not display the deprecated pages.

**BREAKING CHANGE**

n/a

---

## Reviewer Checklist
